maestro/lifecycle.py

```diff
@@ -17,7 +17,7 @@
 class BaseLifecycleHelper:
     """Base class for lifecycle helpers."""
 
-    def test(self):
+    def test(self, ship=None, container=None):
```

Comment:
You don't need to pass both the ship and the container. The container object has a reference to the ship it executes on (`container.ship`)